keyboards/1upkeyboards/super16/config.h

```diff
@@ -21,10 +21,10 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 /* USB Device descriptor parameter */
 #define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x0000
+#define PRODUCT_ID      0x2010
 #define DEVICE_VER      0x0001
-#define MANUFACTURER    1upkeyboards
-#define PRODUCT         super16
+#define MANUFACTURER    1up Keyboards
```

Comment:
I have been working with 1up to get these boards supported by QMK. I've kept the manufacturer name 1upkeyboards across all of them. If you want to make this change, you'll have to change all the boards in 1upkeyboards.